<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>equipe</class>
 <widget class="QWidget" name="equipe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1254</width>
    <height>806</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="TABLE">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1221</width>
     <height>701</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 0, 0);</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_19">
    <attribute name="title">
     <string>Ajouter/supprimer equipe</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>351</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="title">
      <string>Ajouter votre equipe</string>
     </property>
     <widget class="QLabel" name="label_73">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ajouter_nom_equipe">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>100</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter_equipe">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>230</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_74">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ajouter_id_equipe">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_75">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Nom_entraineur</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ajouter_nom_en_equipe">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>160</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="supprimer_equipe">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Supprimer par identifiant</string>
     </property>
    </widget>
    <widget class="QTableView" name="table_equipe">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>70</y>
       <width>771</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="supprimer_equipe_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>380</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="tri_desc_eq">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>20</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Decroissant</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="tri_asc_eq">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>20</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Croissante</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_22">
    <attribute name="title">
     <string>Modifier equipe</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_22">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>351</width>
       <height>341</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="title">
      <string>Modifier votre equipe</string>
     </property>
     <widget class="QLabel" name="label_86">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="modifier_nom_equipe">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>100</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier_equipe">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>250</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_87">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="modifier_id_equipe">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="modifier_nom_en_equipe">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>150</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_88">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Nom_entraineur</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="chercher_equipe">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>60</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="chercher_modif_equipe">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>60</y>
       <width>221</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Chercher equipe par identifiant</string>
     </property>
    </widget>
    <widget class="QTableView" name="table_Re_equipe">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>110</y>
       <width>771</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
